src/main/java/uk/gov/indexer/IndexerTask.java

```diff
@@ -26,12 +36,9 @@ public void run() {
     }
 
     protected void update() {
-        try {
-            int currentWaterMark = destinationDB.currentWaterMark();
-            ResultSet difference = sourceDB.read(currentWaterMark);
-            destinationDB.write(difference);
-        } catch (SQLException e) {
-            throw new RuntimeException(e);
-        }
+        int currentWaterMark = destinationDBQueryDAO.currentWaterMark();
+        //write while loop
```

Comment:
sorry, it is a TODO which I handled in next commit in separate branch. We are currently call update method and then wait for 10 seconds. We should continue update if there are more entries available, for which we should write while loop.
Let me remove this comment from PR.